### ISSUE TITLE
test_capabilities fails in ubuntu/trusty

### ISSUE THREAD

**Issue by elfiesmelfie** [collectd/collectd#3940]
*   Version of collectd: main
*   Operating system / distribution: Ubuntu 14.04 (Trusty)
*   Kernel version (if applicable): n/a

## Expected behavior

make check passes in the debian_default_toolchain job on trusty

## Actual behavior

make check fails due to errors in test_capabilities

-----
Some initial comments are noted in on various PRs (https://github.com/collectd/collectd/pull/3927, https://github.com/collectd/collectd/pull/3936) where this was noticed.

Namely
*  the issue is due to the version of glibc [https://github.com/collectd/collectd/pull/3927#issuecomment-953350598]
* the package is not likely to be updated in trusty
* We can remove trusty from the CI

The capabilities plugin still needs to be tested on ubuntu/trusty, to determine whether the issue is with the functionality of the plugin on this distro, or whether it is likely due to the test.

If the issue effects the functionality of the capabilities plugin, then the package dependencies need to be reviewed/updated to stop it building where this is an issue.

**Comment by elfiesmelfie** [comment]
I tested the capabilities plugin on trusty, the information is collected and consistent with the output from dmidecode.

So there must be an issue in the test, rather than the plugin itself.

**Comment by minfrin** [comment]
This patch needs to be applied to each PR with failures to work around this issue:

```
Index: .cirrus.yml
===================================================================
--- .cirrus.yml	(revision 12535)
+++ .cirrus.yml	(working copy)
@@ -37,7 +37,7 @@
       allow_failures: false
     - container:
         image: collectd/ci:trusty_amd64
-      allow_failures: false
+      allow_failures: true
     - container:
         image: collectd/ci:xenial_amd64
       allow_failures: false
@@ -219,3 +219,4 @@
     - make -j2 -sk
   tests_script:
     - make -j2 -sk check
+
```

**Comment by minfrin** [comment]
Affected PRs:

https://github.com/collectd/collectd/issues/3940
https://github.com/collectd/collectd/pull/3927
https://github.com/collectd/collectd/pull/3859

**Comment by mrunge** [comment]
I disagree with the proposed "workaround", since it will only hide the issue.

**Comment by minfrin** [comment]
> I disagree with the proposed "workaround", since it will only hide the issue.

How do we unblock the project?

We fixed things over a year ago so that secure snmp worked for the first time, ubuntu and Redhat won’t patch anything until these are accepted upstream, and we’ve been blocked behind broken CI ever since.

What is the way forward?
